script/tasks/diamondSyncSigs.sh

```
@@ -23,124 +30,159 @@ function diamondSyncSigs {
   if [[ "$NETWORK" == "All (non-excluded) Networks" ]]; then
     # get array with all network names
     NETWORKS=($(getIncludedNetworksArray))
   else
-    NETWORKS=($NETWORK)
+    NETWORKS=("$NETWORK")
   fi
 
-    # get diamond address from deployments script
-  DIAMOND_ADDRESS=$(getContractAddressFromDeploymentLogs "$NETWORK" "$ENVIRONMENT" "$DIAMOND_CONTRACT_NAME")
+  # Determine file suffix based on environment
+  FILE_SUFFIX=$(getFileSuffix "$ENVIRONMENT")
```

Shellcheck (0.10.0)

[warning] FILE_SUFFIX appears unused. Verify use (or export if used externally).

(SC2034)